services/apps/alcs/src/alcs/search/search.controller.ts

```diff
@@ -288,7 +288,8 @@ export class SearchController {
       !searchDto.dateDecidedTo &&
       !searchDto.resolutionNumber &&
       !searchDto.resolutionYear &&
-      !isStringSetAndNotEmpty(searchDto.legacyId);
+      !isStringSetAndNotEmpty(searchDto.legacyId) &&
+      !isStringSetAndNotEmpty(searchDto.civicAddress);
```

Comment:
question: Why are you changing this? I don't see it mentioned in the MR

Reply:
had to remove it from mr